release-notes/EPM/I-DOCSIS/EPM_7.0.2_I-DOCSIS.md

```diff
@@ -0,0 +1,34 @@
+---
+uid: EPM_7.0.2_I-DOCSIS
+---
+
+# EPM 7.0.2 I-DOCSIS
+
+## New features
+
+#### Harmonic CableOS: New PTP page [ID_39310]
+
+A new PTP page has been added to the Harmonic CableOS connector. On this page, the following new parameters are available:
+
+- CCAP PTP Clock State
+- RPD Port PTP Status Table
+- RPD PTP Clock Status Table
+
+#### New map dashboards added [ID_39319]
```

Comment:
@SkylineGBO, what is the name of these new dashboards? Should we add these on the [I-DOCSIS components page](https://docs.dataminer.services/user-guide/Standard_Apps/EPM/EPM_I-DOCSIS/I-DOCSIS_components.html#dashboards) as well?

Reply:
you are absolutely correct, I have updated the RN with the dashboard name and have updated the components page as well as it was a little outdated with the most recent components. Thank you!